### PR TITLE
Support kilometers and miles for unit internationalization

### LINKED ISSUE #20: Allow using different units of distance
km/mi, etc.

### PR DESCRIPTION
**Pull request by ErikBoesen**
I added code to create a new "Units" option, which allows you to choose between "Metric" and "Imperial" units of measurement. This changes the readout that appears for distances between mi and km.

It is quite possible that I implemented this in a bad way. Please let me know if I need to fix anything.

Fix #20 

I also cleaned up unused imports and ran Android Studio's auto-formatter.

### CODE REVIEW

File: app/src/main/java/com/adisa/diningplus/MainListAdapter.java [hunk 1]
```diff
@@ -101,20 +106,26 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
 
             DecimalFormat numberFormat = new DecimalFormat("0.00");
-            if (item.distance > 50){
-                viewHolder.distance.setText("> 50 mi");
+            String unit = "km";
+            double distance = item.distance;
+            if (!preferences.getString("unitPrefs", "").equals("Imperial")) {
+                unit = "mi";
+                distance *= 0.621371;
+            }
```
Comment (ErikBoesen) [inline app/src/main/java/com/adisa/diningplus/MainListAdapter.java]: Might this logic be better implemented with a switch/case statement?

File: app/src/main/res/xml/preferences_main.xml [hunk 2]
```diff
@@ -9,4 +9,10 @@
     <com.adisa.diningplus.FollowDialogPreference
         android:summary="Receive notifications when these items are being served."
         android:title="Followed items" />
+    <ListPreference
+        android:entries="@array/units"
+        android:entryValues="@array/units"
+        android:key="unitPrefs"
```
Comment (amalik12) [inline app/src/main/res/xml/preferences_main.xml]:
```suggestion
        android:defaultValue="Imperial"
        android:key="unitPrefs"
```
Set Imperial to be selected by default when the user loads the preference screen

File: app/src/main/java/com/adisa/diningplus/MainListAdapter.java [hunk 3]
```diff
@@ -101,20 +106,26 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
+            double distance = item.distance;
+            if (preferences.getString("unitPrefs", "").equals("Imperial")) {
+                unit = "mi";
+                distance *= 0.621371;
+            }
+            if (distance > 50) {
+                viewHolder.distance.setText("> 50" + unit);
             } else {
-                viewHolder.distance.setText("" + numberFormat.format(item.distance) + " mi");
+                viewHolder.distance.setText("" + numberFormat.format(distance) + unit);
```
[the remark this answers is not in the document]
Comment (ErikBoesen) [reply app/src/main/java/com/adisa/diningplus/MainListAdapter.java]: Did a similar thing in a different way

**Comment by ErikBoesen** [comment]
Fixed

**Pull request merged**